docs/source/reference/_create_conda_env.rst

```diff
@@ -0,0 +1,383 @@
+    additional channels to search in using the <code>-c</code> or
+    <code>--channel</code> option when using the conda install command.
+
+    </details>
+
+
+    <details>
+    <summary>libmamba solver</summary>
+
+    In 2022, a <a
+    href="https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community">significant
+    performance enhancement</a> was made available to conda by inclusion of the
+    <a
+    href="https://conda.github.io/conda-libmamba-solver/">conda-libmamba-solver</a>
+    package. If you have installed miniconda using the instructions for Linux
+    above, this package is already installed and configured as the default solver.
+    
+    You can check your default solver with the following command:
+
+    <pre>
+    $ <b>conda config --show solver</b>
+    </pre>
+
+    If the above command returns <code>classic</code>, follow those steps:
+
+    <ul>
+    <li>Install the libmamba solver:</li>
+    <pre>
+    $ <b>conda install -c conda-forge libmamba</b>
+    </pre>
+    <li>Set libmamba as the default solver:</li>
+    <pre>
+    $ <b>conda config --set solver libmamba</b>
+    </pre>      
+    <li>Confirm libmanda is now your default solver:</li>
+    <pre>
+    $ <b>conda config --show solver</b>
+    </pre>
+    </ul>
+
+    </details>
+
+
+
+
+Install the bluesky environment
+-------------------------------
+
+The following commands install the ``bluesky_2023_2`` environment inside the
+``bluesky`` directory  that was created when installing a new bluesky instrument
+(see `setup a bluesky instrument
+<https://bcda-aps.github.io/bluesky_training/instrument/_install_new_instrument.html#setup-a-bluesky-instrument>`__).
+
+
+
+Note that the installation takes several minutes.
+
+.. raw:: html
+
+    <pre>
+    $ <b>cd ~/bluesky</b>
+    $ <b>conda env create \
+        --force \
+        -n bluesky_2023_2 \
+        -f ./environments/environment_2023_2.yml \
+        --solver=libmamba</b>
+    </pre>
+
+    <details><summary>Details</summary>
+    
+    <ul> 
+    <li>The <code>--force</code> option will replace any existing environment by this
+    name without asking for confirmation. Remove this option if you wish.</li>
+    <li>The <code>-n bluesky_2023_2</code> sets the name of the conda environment to be
+    created.</li> 
+    <li>The <code>-f ./environments/environment_2023_2.yml</code> option names the 
+    YAML file to be used. We create different versions of the YAML file, named for the APS operating cycle (2021-1, 2023-2, …), as the suite of packages for a working installation may change over time. By keeping all these files in the environments subdirectory, we can restore any of these environments with a simple command.</li> 
+    <li>The <code>--solver=libmamba</code> option will use the conda-libmamba-solver. This option is optional (can be removed) but result in a much faster installation of the bluesky environment. The libmanba installation is described in the previous section.</li> 
```

Comment:
change:  This option is optional (can be removed) but result in a much faster installation of the bluesky environment.

to: The `--solver` option can be removed but its use results in a much faster installation of the bluesky environment.

Reply:
change `libmanba` to `libmamba`